server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/dto/FilterRuleConditionDTO.java

```diff
@@ -0,0 +1,98 @@
+/****************************************************************
```

Comment:
What are these changes about JMAP filtering doing in Webdmin?

https://github.com/linagora/james-project/pull/1679/files/952af498a9ef2b1490bc80ccb1e4cfda4d434493..7a3c048f6100afc432d6360ffbd8d4d7d3393c7f